OH-SHOWN/ohshown-backend
# Update ohshown_event model and add "sight-seeing date and time" in the django admin console

This pull request aims at 2 missiones:
1. Update the ohshown_event model in the "standard" django way. See the deprecated pull request #58 
2. Add "sight-seeing date and time" in the django admin console for issue #34

After landing this pull request, you should see the django admin console looks like this:

![Selection_050](https://user-images.githubusercontent.com/3217432/164988693-df351e83-eaa4-4152-aecb-cde83f010f75.png)

# Steps to Verify
1. Rebuild everything `docker-compose -f docker-compose.dev.yml down --rmi all --volumes; sudo rm -rf /tmp/disfactory_d*; docker-compose -f docker-compose.dev.yml up -d; sleep 15; docker-compose exec web python manage.py createsuperuser ;docker-compose logs --tail 100 -f web`
2. Go to the Django admin console with the credential created at step1
3. Check the `api/ohshown_event` in the admin console.

# Expected Result
1. See the attached image above.
2. When creating new entries via the frontend, you should see new entries in the admin console as well.
